formatter.go

```diff
@@ -677,30 +712,49 @@ var defaultTemplateFuncs = template.FuncMap{
 
 		return sb.String()
 	},
+	"color": func(enable bool, c, s string) string {
+		if !enable {
+			return s
+		}
+		var ca color.Attribute
+		switch c {
+		case "red":
+			ca = color.FgRed
+		case "yellow":
+			ca = color.FgYellow
+		case "green":
+			ca = color.FgGreen
+		case "cyan":
+			ca = color.FgCyan
+		default:
+			ca = color.FgCyan
+		}
+		return color.New(ca).Sprint(s)
+	},
 }
 
-var defaultSuccessTemplate = `[OK] {{ join .AssertPath .LineWidth }}`
+var defaultSuccessTemplate = `[OK] {{ join .AssertPath .LineWidth | color .EnableColor "green" }}` //nolint
```

Comment:
I think for success template we remove coloring. Otherwise the screen will be full of green lines :)